src/frontend/src/pages/PIAForm/BannerStatus/messages.ts

```diff
@@ -7,6 +7,14 @@ export default {
       en: `To submit this PIA to the CPO for review, download it as a PDF and send it to the CPO via email.`,
     },
   },
+  CPOReviewStatusCalloutText: {
+    Drafter: {
+      en: `Your PIA is being reviewed by the Corporate Privacy Office (CPO). CPO or your MPO will contact you with any questions.`,
```

Comment:
We should clarify with POs about using "CPO or..." vs "The CPO or..." and similar for other uses of the acronym. WIthout the 'The' it reads, "Corporate Privacy Office or your MPO will contact...", which is grammatically weird.